lib/common_test/src/ct_master.erl

```diff
@@ -557,7 +557,7 @@ refresh_logs([D|Dirs],Refreshed) ->
 		    refresh_logs(Dirs,Refreshed);
 		false ->
 		    {ok,Cwd} = file:get_cwd(),
-		    case catch ct_run:refresh_logs(D) of
+		    case catch ct_run:refresh_logs(D, unknown) of
```

Comment:
would it make code less ugly if unknown is a default when ct_run:refresh_logs/1 ?
maybe ct_master module can be left alone, not being aware about stylesheet?

Reply:
see my response above